MicrosoftTeams.schema.json

```diff
@@ -805,6 +805,24 @@
       },
       "description": "When a group install scope is selected, this will define the default capability when the user installs the app",
       "additionalProperties": false
+    },
+    "configurableProperties": {
+      "type": "array",
+      "description": "A list of tenant configurable properties for an app",
+      "maxItems": 9,
+      "items": {
+        "enum": [
+          "name",
+          "shortDescription",
+          "longDescription",
+          "smallImageUrl",
+          "largeImageUrl",
+          "accentColor",
+          "developerUrl",
```

Comment:
What does developerUrl map to?

Reply:
developerUrl maps to the developer.websiteUrl